lambda/config.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
@jgigler - is the intention that config.json be not present? I want to include it so that anyone can just change the lambda folder alone and zip and deploy that without even using the terraform scripts. What do you think?